docs/change_log/2_0_0_rc2.md

```diff
@@ -3,9 +3,11 @@
 ```eval_rst
 .. admonition:: v2.0.0-rc1升级到v2.0.0-rc2
 
-    - **兼容升级** ：直接替换v2.0.0-rc1节点的二进制为v2.0.0-rc2二进制，升级后的版本修复v2.0.0-rc1中的bug，但不会启用v2.0.0-rc2并行计算、分布式存储等新特性， **升级到v2.0.0-rc2后，无法回滚到v2.0.0-rc1**
+    - **兼容升级** ：直接替换v2.0.0-rc1节点的二进制为 `v2.0.0-rc2二进制 <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v2.0.0-rc2/fisco-bcos.tar.gz>`_ ，升级后的版本修复v2.0.0-rc1中的bug，但不会启用v2.0.0-rc2并行计算、分布式存储等新特性， **升级到v2.0.0-rc2后，无法回滚到v2.0.0-rc1**
 
     - **全面升级** ：参考 `安装 <../installation.html>`_ 搭建新链，重新向新节点提交所有历史交易，升级后节点包含v2.0.0-rc2新特性
+
+    - `v2.0.0-rc2 release <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/tag/v2.0.0-rc2>`_
```

Comment:
v2.0.0-rc2 release notes

Reply:
already modified from `release note` to `Relase Note`